frontend/src/App.jsx

```diff
@@ -6,15 +6,15 @@ import Home from "@pages/Home";
 
 function App() {
   return (
-   <div className="App">
-    <Routes>
-      <Route path="/search" element={<Search />} />
-      <Route path="/" element={<Home />} />
-      <Route path="/profile" element={<Profile />} />
-      <Route path="/profile/:profileId" element={<Profile />} />
-    </Routes>
-    <Navbar />
-  </div>
+    <div className="App">
+      <Routes>
+        <Route path="/search" element={<Search />} />
+        <Route path="/" element={<Home />} />      
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/profile/:profileId" element={<Profile />} />
+      </Routes>
+      <Navbar />
+    </div>
```

Comment:
Why is this file touches at all ?
On the plus side, you seem to fix indentation
On the minus side, you create a useless empty space...
Prettier didn't go through this, did it ?